servers/intro_course/developerAccountSetup/apple.go

```
@@ -0,0 +1,179 @@
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return storeAppleError(ctx, coursePhaseID, participationID, fmt.Errorf("request failed: %w", err))
+	}
+	defer resp.Body.Close()
+
+	body, _ := io.ReadAll(resp.Body)
+	if resp.StatusCode != 201 {
+		err := fmt.Errorf("failed to register device: %s, details: %s", resp.Status, string(body))
+		storeAppleError(ctx, coursePhaseID, participationID, err)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)